### PR TITLE
Adding select menu for sorting results on search results page

### PR DESCRIPTION
**Pull request by matt-blair**
Hi @Terry-Price,

Would you mind have a quick check of this PR?

Changes are:

- Adding a select menu and submit button to sort the results on the search results page
- Amend the corresponding label to say 'Order results by'

Let me know if there are any problems.

Matt.

### CODE REVIEW

File: app/static/styles/src/sass/includes/_result_controls.scss [hunk 1]
```diff
@@ -1,14 +1,38 @@
 .result-controls {
+  text-align: left;
+  display: inline-block;
+  margin: 1rem 0 0;
```
Comment (Terry-Price) [inline app/static/styles/src/sass/includes/_result_controls.scss]: just wondered if 1rem was a variable too.
not sure if we use it here here?

File: app/static/styles/src/sass/includes/_result_controls.scss [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+  @media (min-width: $grid__breakpoint-extra-large) {
+    text-align: right;
+    margin: 0;
+  }
+
 
   &__order-by-label {
     font-family: $font__roboto;
-    font-size: 1.4rem;
+    font-size: 1rem;
```
Comment (Terry-Price) [inline app/static/styles/src/sass/includes/_result_controls.scss:14]: same comment as above.

**Review by Terry-Price — COMMENTED** [review]
jst wondered if variable was used for 1rem?

**Comment by matt-blair** [comment]
Hi @Terry-Price. Thanks for checking these. I've changed the colour values to use variables and also the margin value you spotted to use the spacer__unit variable. I think it's probably best to only use that for margin and padding and just use specific rem values for font-size so we don't have any dependency on variables across fonts and spacing.

**Pull request merged**